main_routers/characters_router.py

```
@@ -330,33 +359,181 @@ async def update_catgirl_l2d(name: str, request: Request):
+
+        # 保存配置到文件
+        _config_manager.save_characters(characters)
+        
+        # 条件性完整重载：只在明确请求时执行（避免频繁重载）
+        if apply_runtime:
+            initialize_character_data = get_initialize_character_data()
+            if initialize_character_data:
+                await initialize_character_data()
+                logger.info(f"已执行完整配置重载（角色 {name} 的打光配置）")
+        else:
+            logger.debug(f"跳过完整配置重载（apply_runtime=False），配置已保存到磁盘，需要刷新页面或调用重载才能生效")
```

Ruff (0.14.10)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)